pages.it/common/git-check-ignore.md

```diff
@@ -0,0 +1,24 @@
+# git check-ignore
+
+> Analizza ed esegui il debug di ".gitignore" e dei file esclusi.
+> Maggiori informazioni: <https://git-scm.com/docs/git-check-ignore>.
+
+- Verifica se un file o una cartella sono ignorati:
+
+`git check-ignore {{percorso/al/file_o_cartella}}`
+
+- Verifica se più file o cartelle sono ignorati:
+
+`git check-ignore {{percorso/al/file}} {{percorso/alla/cartella}}`
+
+- Leggi i percorsi di file o cartelle da `stdin` (uno per riga) invece che dalla riga di comando:
+
+`git check-ignore --stdin < {{percorso/alla/lista_dei_file_o_cartelle}}`
+
+- Non controllare nell'indice (usato per il debug del perché alcuni percorsi non sono ignorati):
+
+`git check-ignore --no-index {{percorsi/ai/file_o_cartelle}}`
+
+- Includi dettagli sui modelli di corrispondenza per ogni percorso specificato:
```

Comment:
```suggestion
- Includi dettagli sul pattern corrispondente per ogni percorso specificato:
```
Same.

Reply:
this is exactly why I wanted a preliminary PR :D I didn't know how much to translate. Thanks again for your effort, @mebeim